openquake/commonlib/calculators/event_based_risk.py

```diff
@@ -151,25 +174,28 @@ def post_execute(self, result):
         several interesting outputs.
         """
         oq = self.oqparam
+        self.rlzs_assoc = self.rlzs_assoc  # write anew
```

Comment:
It takes the attribute `self.rlzs_assoc` from the cache (where it may have been updated) and write it again on the datastore.